corehq/apps/cleanup/management/commands/couch_integrity.py

```diff
@@ -110,16 +109,22 @@ def print_result(matches, view, database):
     if not matches:
         return
 
+    COUCH = 0
+    ROWS = 1
+
     if len(matches) == 1:
-        print u"All is consistent in {} for view {}".format(database, view)
+        print u"{}All is consistent in {} for view {}{}".format(Colors.OKGREEN, database, view, Colors.ENDC)
         return
 
-    for rows, databases in matches.items():
-        print "##### {} #####".format(view)
-        print u"Databases: "
-        for db in databases:
-            print db
-        print u"Had this many {} rows for this view".format(rows)
+    print "{}{} - {}{}".format(Colors.WARNING, database, view, Colors.ENDC)
+    for wiggle_range, match_tuples in matches.items():
+        print u"Couches for wiggle range {}: ".format(wiggle_range)
+        for match_tuple in match_tuples:
```

Comment:
you can replace this line with
```python
for couch_uri, total_rows in match_tuples:
```
to unpack the tuple in-place, rather than doing it by hand later.